sdk/src/main/java/io/customer/sdk/CustomerIO.kt

```diff
@@ -198,6 +221,14 @@ class CustomerIO internal constructor(
         deviceToken = deviceToken
     )
 
+    /**
+     * Use to provide additional and custom device attributes
+     * apart from the ones the SDK is programmed to send to customer workspace.
+     */
+    fun addCustomDeviceAttributes(attributes: Map<String, Any>) {
```

Comment:
Majorly because of in `Java` it will not look like this, it would look something like this

```
 CustomerIO.instance().getDeviceAttributes().put("", "");
```

Also, `deviceAttributes` would be `final`. But I can try parity with iOS.

Reply:
I had a feeling it was for Java compatability. 

I would love to see our SDK be Kotlin-first with java support added on top. Make the SDK's API feel Kotlin-y. Right now it feels like our SDK is designed to be a Java SDK that happens to work with Kotlin with some nice Kotlin features built-in such as coroutines. 

This is my opinion and something we can address more long-term if we choose. Highest priority today is our SDK has Java support and you have done a great job making sure that is the case.